blob/blob-aws/src/test/java/com/salesforce/multicloudj/blob/aws/AwsBlobBenchmarkTest.java

```diff
@@ -0,0 +1,134 @@
+import org.slf4j.LoggerFactory;
+
+@Disabled
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class AwsBlobBenchmarkTest extends AbstractBlobBenchmarkTest {
+
+    private static final Logger logger = LoggerFactory.getLogger(AwsBlobBenchmarkTest.class);
+    private static final String endpoint = "https://s3.us-east-2.amazonaws.com";
+    private static final String bucketName = "chameleon-jcloud-benchmarks";
+    private static final String versionedBucketName = "chameleon-jcloud-versioned";
+    private static final String nonExistentBucketName = "java-bucket-does-not-exist";
+    private static final String region = "us-east-2";
+
+    @Override
+    protected Harness createHarness() {
+        return new HarnessImpl();
+    }
+
+    @Override
+    @Test
+    public void runBenchmarks() throws RunnerException {
+        Options opt = new OptionsBuilder()
+                .include(".*benchmarkSingleActionPut.*")
+                //.include(".*" + this.getClass().getName() + ".*") 
+                .forks(1)
+                .warmupIterations(3)
+                .measurementIterations(5)
+                .resultFormat(org.openjdk.jmh.results.format.ResultFormatType.JSON)
+                .result("target/jmh-results.json")
+                .build();
+
+        new Runner(opt).run();
+    }
+
+    public static class HarnessImpl implements Harness {
+        SdkHttpClient httpClient;
+        S3Client client;
+
+        @Override
+        public AbstractBlobStore<?> createBlobStore(boolean useValidBucket, boolean useValidCredentials, boolean useVersionedBucket) {
+            String accessKeyId = System.getenv().getOrDefault("AWS_ACCESS_KEY_ID", "FAKE_ACCESS_KEY");
+            String secretAccessKey = System.getenv().getOrDefault("AWS_SECRET_ACCESS_KEY", "FAKE_SECRET_ACCESS_KEY");
+            String sessionToken = System.getenv().getOrDefault("AWS_SESSION_TOKEN", "FAKE_SESSION_TOKEN");
+
+            StsCredentials sessionCreds = new StsCredentials(accessKeyId, secretAccessKey, sessionToken);
+            CredentialsOverrider credentialsOverrider = new CredentialsOverrider.Builder(CredentialsType.SESSION)
+                    .withSessionCredentials(sessionCreds).build();
+            
+            String bucketNameToUse = useValidBucket ? 
+                    (useVersionedBucket ? versionedBucketName : bucketName) : 
+                    nonExistentBucketName;
+
+            return createBlobStore(bucketNameToUse, credentialsOverrider);
+        }
+
+        private AbstractBlobStore<?> createBlobStore(final String bucketName, final CredentialsOverrider credentialsOverrider) {
+            AwsSessionCredentials awsCredentials = AwsSessionCredentials.create(
+                    credentialsOverrider.getSessionCredentials().getAccessKeyId(),
+                    credentialsOverrider.getSessionCredentials().getAccessKeySecret(),
+                    credentialsOverrider.getSessionCredentials().getSecurityToken());
+
+
```

Comment:
i would suggest removing these at all and let AWS use the default credentials based on env vars